app/models/concerns/likeable.rb

```diff
@@ -0,0 +1,31 @@
+module Likeable
+  extend ActiveSupport::Concern
+
+  include AccountAPIHelper
+
+  class_methods do
+    def add_likeable_for
+      define_method "like" do |user_id|
+        unless find_like(user_id)
+          Like.create(user_id: user_id, target: self)
+          { message: 'success'}
```

Comment:
A model method should just return `true/false` here, instead of a message. Let the controllers do the job.